SU-SWS/saa_victoria_subtheme
# ADAPT-476 Fixed nav hover issue

# READY FOR REVIEW

# Summary
- Fixed nav hover issue

# Review By (Date)
- asap

# Urgency
- high

# Steps to Test
1. Pull this branch
2. Clear cache
3. Switch to another menu item (that is not the Home). Hover over the active menu item. The border should now change on hover instead of disappearing.

# Associated Issues and/or People
- ADAPT-476

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
